### PR TITLE
INS-19692 - kafka version 3.1.1 ga

### PR DESCRIPTION
**Pull request by vutdv-ic**
add reference to kafka 3.1.1 and kafka connect 3.1.1
update example to latest version

### CODE REVIEW

**Review by Mitchel-Stig-Instaclustr — DISMISSED** [review]

File: examples/main.tf [hunk 1]
```diff
@@ -542,7 +542,7 @@ resource "instaclustr_cluster" "example-cadenceav-kafka" {
 
   bundle {
     bundle = "KAFKA"
-    version = "3.0.0"
+    version = "3.1.1"
```
Comment (YuWang24) [inline examples/main.tf:545]: There's no need to change all the versions to 3.1.1. Especially for Cadence, if we want to change this, we'd better to test it whether this version can work well with Cadence.

Comment (sujayinstaclustr) [reply examples/main.tf:545]: @vutdv-ic  @YuWang24  is right Cadence has a pinned version of Kafka (3.0.0) which is a hard dependency on the Kafka version. Don't change this unless Cadence team wants to change it.

Comment (vutdv-ic) [reply examples/main.tf]: right, updated

File: Makefile [hunk 2]
```diff
@@ -5,7 +5,7 @@ BIN_NAME=terraform-provider-instaclustr
 # for VERSION, don't add prefix "v", e.g., use "1.9.8" instead of "v1.9.8" as it could break circleCI stuff
 
 
-VERSION=1.25.0
+VERSION=1.26.0
```
Comment (arjunrajnetapp) [inline Makefile:8]: Should be a patch version bump as per semver rules as this is pretty much just documentation changes and test fixes, and we aren't adding any new features.

**Review by arjunrajnetapp — CHANGES_REQUESTED** [review]

**Review by arjunrajnetapp — DISMISSED** [review]

**Review by sujayinstaclustr — DISMISSED** [review]

**Review by YuWang24 — DISMISSED** [review]

**Review by YuWang24 — APPROVED** [review]

**Review by arjunrajnetapp — APPROVED** [review]

**Pull request merged**